### DISCUSSION TITLE
GEO Coding "Am Wassen"

### DISCUSSION THREAD

**Discussion by Andy-Schulte** [komoot/photon#676] [Q&A]
in the API search for
Am Wassen 11, 4755 Zell an der Pram in Austria will show 3 results in this order.
1. Am Wassen-Süd 11
2. Am Wassen-Nord 11
3. Am Wassen 11
but Am Wassen 11 should be the first and only result.

What can I do to get the correct result?

**Comment by lonvia** [comment]
This would need a boost for results with complete matches. This is a bit of a pain to do with ElasticSearch.
